packages/database/prisma/schema.prisma

```diff
@@ -202,3 +207,63 @@ model Incentive {
   @@index(fields: [rewardTokenId])
   @@index(fields: [chainId, rewardTokenId])
 }
+
+enum SteerStrategy {
+  DeltaNeutralStables
+  ElasticExpansion
+  MovingVolatilityChannelMedium
+  HighLowChannel
+  StaticStable
+  ClassicRebalance
+}
+
+enum VaultState {
+  PendingApproval
+  PendingThreshold
+  Paused
+  Active
+  Retired
+}
+
+model SteerVault {
+  id                      String      @id // chainId:address
+  
+  pool                    SushiPool   @relation(fields: [poolId], references: [id])
+  poolId                  String
+  feeTier                 Float
+  
+  apr                     Float
+  apr1d                   Float
+  apr1m                   Float
+  apr1y                   Float
+
+  token0                  Token       @relation("token0", fields: [token0Id], references: [id])
```

Comment:
My expectation is that strategies will be created for existing pools only (you need to pass a pool address to create a vault onchain, and I expect the pool to be indexed already).

So yeah it's optimistic, what would happen if the token didn't exist yet? If it'd break the whole thing, guess we should make sure it's there...

Reply:
Don't know if there are any constraints on the db for it.. 

If there's a maintenance window on hosted service and we don't get a pool right away, this could get into a bad state, should probably check if they exist?